dita command doesn't work!

Following is the result log:

PS D:\DITA-OT\dita-ot-3.5.2> bin/dita -i D:\SVN\pdf5\testdata\20200514-track-change\bmTrackChanges3.ditamap -f pdf5.ml -o D:\SVN\pdf5\testdata\20200514-track-change\out --ahf.dir "C:\Program Files\Antenna House\AHFormatterV70" --ahf.opt "C:\Program Files\Antenna House\AHFormatterV70\ahf_setting.xml" --output.changes.and.comments yes --output.changebars yes --verbose
Unable to locate tools.jar. Expected to find it in C:\Program Files\Java\jre1.8.0_261\lib\tools.jar
Searching for integrator.xml ...
     [echo] com.antennahouse.i18n_index.2.4 is integrated.

init:
     [echo] com.antennahouse.i18n_index.2.4 is integrated.

init-properties:
[init-project] Initializing project

check-arg:
    [mkdir] Created dir: C:\Users\toshi\AppData\Local\Temp\temp20200819012120366

log-arg:
     [echo] *****************************************************************
     [echo] * basedir = D:\DITA-OT\dita-ot-3.5.2
     [echo] * dita.dir = D:\DITA-OT\dita-ot-3.5.2\bin\..
     [echo] * transtype = pdf5.ml
     [echo] * tempdir = C:\Users\toshi\AppData\Local\Temp\temp20200819012120366
     [echo] * outputdir = D:\SVN\pdf5\testdata\20200514-track-change\out
     [echo] * clean.temp = true
     [echo] * DITA-OT version = 3.5.2
     [echo] * XML parser = Xerces
     [echo] * XSLT processor = Saxon
     [echo] * collator = ICU
     [echo] *****************************************************************
     [echo] #Ant properties
     [echo] #Wed Aug 19 01:21:20 JST 2020
     [echo] args.grammar.cache=yes
     [echo] args.input=D\:\\SVN\\pdf5\\testdata\\20200514-track-change\\bmTrackChanges3.ditamap
     [echo] args.xml.systemid.set=yes
     [echo] dita.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\bin\\..
     [echo] dita.output.dir=D\:\\SVN\\pdf5\\testdata\\20200514-track-change\\out
     [echo] dita.plugin.com.antennahouse.dita.dita13.doctypes.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.antennahouse.dita.dita13.doctypes
     [echo] dita.plugin.com.antennahouse.i18n_index.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.antennahouse.i18n_index.2.4
     [echo] dita.plugin.com.antennahouse.odt.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.antennahouse.odt
     [echo] dita.plugin.com.antennahouse.pdf.review.oxygen.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.antennahouse.pdf.review.oxygen
     [echo] dita.plugin.com.antennahouse.pdf5.ml.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.antennahouse.pdf5.ml
     [echo] dita.plugin.com.antennahouse.pdf5.ml.index.extension.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.antennahouse.pdf5.ml.index.extension
     [echo] dita.plugin.com.antennahouse.wml.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.antennahouse.wml
     [echo] dita.plugin.com.aristocrat.dtd.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.aristocrat.dtd
     [echo] dita.plugin.com.elovirta.ooxml.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.elovirta.ooxml
     [echo] dita.plugin.com.oxygenxml.pdf.review.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\com.oxygenxml.pdf.review
     [echo] dita.plugin.org.dita.base.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.base
     [echo] dita.plugin.org.dita.eclipsehelp.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.eclipsehelp
     [echo] dita.plugin.org.dita.html5.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.html5
     [echo] dita.plugin.org.dita.htmlhelp.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.htmlhelp
     [echo] dita.plugin.org.dita.index.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.index
     [echo] dita.plugin.org.dita.normalize.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.normalize
     [echo] dita.plugin.org.dita.pdf2.axf.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.pdf2.axf
     [echo] dita.plugin.org.dita.pdf2.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.pdf2
     [echo] dita.plugin.org.dita.pdf2.fop.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.pdf2.fop
     [echo] dita.plugin.org.dita.pdf2.xep.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.pdf2.xep
     [echo] dita.plugin.org.dita.specialization.dita11.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.specialization.dita11
     [echo] dita.plugin.org.dita.xhtml.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.dita.xhtml
     [echo] dita.plugin.org.lwdita.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.lwdita
     [echo] dita.plugin.org.oasis-open.dita.techcomm.v2_0.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.oasis-open.dita.techcomm.v2_0
     [echo] dita.plugin.org.oasis-open.dita.v1_2.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.oasis-open.dita.v1_2
     [echo] dita.plugin.org.oasis-open.dita.v1_3.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.oasis-open.dita.v1_3
     [echo] dita.plugin.org.oasis-open.dita.v2_0.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.oasis-open.dita.v2_0
     [echo] dita.plugin.org.oasis-open.xdita.v0_2_2.dir=D\:\\DITA-OT\\dita-ot-3.5.2\\plugins\\org.oasis-open.xdita.v0_2_2
     [echo] dita.temp.dir=C\:\\Users\\toshi\\AppData\\Local\\Temp\\temp20200819012120366
     [echo] preprocess.chunk.skip=true
     [echo] preprocess.copy-html.skip=true
     [echo] preprocess.copy-image.skip=true
     [echo] *****************************************************************

build-init:

preprocess.init:
     [echo] *****************************************************************
     [echo] * input = D:\SVN\pdf5\testdata\20200514-track-change\bmTrackChanges3.ditamap
     [echo] * resources = ${args.resources}
     [echo] *****************************************************************

ditaval-merge:

gen-list:
 [gen-list] Using Xerces grammar pool for DTD and schema caching.
 [gen-list] Processing file:/D:/SVN/pdf5/testdata/20200514-track-change/bmTrackChanges3.ditamap
 [gen-list] Processing file:/D:/SVN/pdf5/testdata/20200514-track-change/topic/p_programmingelements.xml
 [gen-list] Serializing job specification

debug-filter:
   [filter] Using Xerces grammar pool for DTD and schema caching.
   [filter] Processing file:/D:/SVN/pdf5/testdata/20200514-track-change/bmTrackChanges3.ditamap to file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/bmTrackChanges3.ditamap
   [filter] Processing file:/D:/SVN/pdf5/testdata/20200514-track-change/topic/p_programmingelements.xml to file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/topic/p_programmingelements.xml

mapref-check:

mapref:
   [mapref] Processing file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/bmTrackChanges3.ditamap

branch-filter:
[branch-filter] Processing file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/bmTrackChanges3.ditamap

keyref:
   [mapref] Transforming into C:\Users\toshi\AppData\Local\Temp\temp20200819012120366
   [mapref] Loading stylesheet D:\DITA-OT\dita-ot-3.5.2\plugins\org.dita.base\xsl\preprocess\mapref.xsl
   [mapref] Processing C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\bmTrackChanges3.ditamap

copy-to:

conrefpush:

conref:

profile-check:

profile:

topic-fragment:
[topic-fragment] Processing file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/topic/p_programmingelements.xml

chunk-check:

chunk:

move-meta-entries:
[move-meta] Processing file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/bmTrackChanges3.ditamap
[move-meta] Processing file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/topic/p_programmingelements.xml
[move-meta] Loading stylesheet D:\DITA-OT\dita-ot-3.5.2\plugins\org.dita.base\xsl\preprocess\mappull.xsl
[move-meta] Processing file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/bmTrackChanges3.ditamap

maplink-check:

maplink:

topicpull-check:

topicpull:
[topicpull] Transforming into C:\Users\toshi\AppData\Local\Temp\temp20200819012120366
[topicpull] Loading stylesheet D:\DITA-OT\dita-ot-3.5.2\plugins\org.dita.base\xsl\preprocess\topicpull.xsl
[topicpull] Processing C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\topic\p_programmingelements.xml

clean-map-check:

clean-map:
[clean-map] Transforming into C:\Users\toshi\AppData\Local\Temp\temp20200819012120366
[clean-map] Loading stylesheet D:\DITA-OT\dita-ot-3.5.2\plugins\org.dita.base\xsl\preprocess\clean-map.xsl
[clean-map] Processing C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\bmTrackChanges3.ditamap

clean-preprocess:

copy-image:

copy-html:

copy-flag-check:

copy-flag:

copy-files:

preprocess:

map2pdf5.ml:
     [echo] Start generating merged middle file.
 [pipeline] Processing C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\bmTrackChanges3.ditamap
 [pipeline] Processing C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\topic\p_programmingelements.xml

merged2pdf:
     [echo] dita.temp.dir=C:\Users\toshi\AppData\Local\Temp\temp20200819012120366 build.temp.dir=C:\Users\toshi\AppData\Local\Temp\temp20200819012120366
     [echo] dita.dir=D:\DITA-OT\dita-ot-3.5.2\bin\..
     [echo] basedir=D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf5.ml
     [echo] outputFile=D:\SVN\pdf5\testdata\20200514-track-change\out\bmTrackChanges3.pdf

build.disp.property:
     [echo] *-- Plug-in Parameters -----------------------------------------*
     [echo] style.def.file=${style.def.file}
     [echo] assume.sortas.pinyin=${assume.sortas.pinyin}
     [echo] make.see.link=${make.see.link}
     [echo] include.frontmatter.to.toc=${include.frontmatter.to.toc}
     [echo] add.numbering.title.prefix=${add.numbering.title.prefix}
     [echo] add.part.to.title=${add.part.to.title}
     [echo] add.thumbnail.index=${add.thumbnail.index}
     [echo] xml.lang=${xml.lang}
     [echo] output.pdf=bmTrackChanges3.pdf
     [echo] output.draft.comment=${output.draft.comment}
     [echo] output.required.cleanup=${output.required.cleanup}
     [echo] use.oid=${use.oid}
     [echo] format.dl.as.block=${format.dl.as.block}
     [echo] debug.style=${debug.style}
     [echo] sort.glossentry=${sort.glossentry}
     [echo] output.start.message=${output.start.message}
     [echo] auto.scale.down.to.fit=${auto.scale.down.to.fit}
     [echo] make.toc.for.map=${make.toc.for.map}
     [echo] make.index.for.map=${make.index.for.map}
     [echo] make.alt.txt=${make.alt.txt}
     [echo] paper.size=${paper.size}
     [echo] image.in.output.folder=no
     [echo] output.crop.region=${output.crop.region}
     [echo] output.type=web
     [echo] exclude.cover.from.counting.page=${exclude.cover.from.counting.page}
     [echo] number.equation.block.unconditionally=${number.equation.block.unconditionally}
     [echo] exclude.auto.numbering.from.equation.figure=${exclude.auto.numbering.from.equation.figure}
     [echo] assume.equation.number.as.auto=${assume.equation.number.as.auto}
     [echo] debug.index.sort.result=${debug.index.sort.result}
     [echo] skip.pdf.generation=no
     [echo] output.index=${output.index}
     [echo] display.fn.at.end.of.topic=${display.fn.at.end.of.topic}
     [echo] ahf.dir=C:\Program Files\Antenna House\AHFormatterV70
     [echo] ahf.opt=C:\Program Files\Antenna House\AHFormatterV70\ahf_setting.xml
     [echo] user.input.dir.url=file:/D:/SVN/pdf5/testdata/20200514-track-change/
     [echo] dita.map.filename.root=bmTrackChanges3
     [echo] com.innodata.i18n.home=D:\DITA-OT\dita-ot-3.5.2\bin\..\plugins\com.antennahouse.i18n_index.2.4\lib\i18n_support
     [echo] use.i18n.index.lib=no
     [echo] use.saxon.transform=no
     [echo] *----------------------------------------------------------------*

transform.merged2pdf.impl:

transform.convmerged.pre.processing:
     [echo] [com.antennahouse.pdf.review.oxygen] target: transform.convmerged.pre.processing has been invoked.

transform.convmerged.pre.processing.impl:
     [xslt] Processing C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\bmTrackChanges3_MERGED.xml to C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\bmTrackChanges3_MERGED_PRE_1.xml
     [xslt] Loading stylesheet D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf.review.oxygen\xsl\dita2fo_change_tracking_shell.xsl
     [xslt] Building tree for file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/bmTrackChanges3_MERGED.xml using class net.sf.saxon.tree.tiny.TinyBuilder
     [xslt] Tree built in 2.1103ms
     [xslt] Tree size: 475 nodes, 7083 characters, 368 attributes
     [xslt] Using parser org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser
     [xslt] Writing to file:/C:/Users/toshi/AppData/Local/Temp/temp20200819012120366/dita-merge.topic1-3.xml
     [copy] Copying 1 file to C:\Users\toshi\AppData\Local\Temp\temp20200819012120366
     [move] Moving 1 file to C:\Users\toshi\AppData\Local\Temp\temp20200819012120366

transform.merged.impl:

transform.merged.common:

build.project.class.path.wo.plugin:

build.project.class.path.with.plugin:

build.project.class.path:

transform.merged.common.saxon:

build.project.class.path.wo.plugin:

build.project.class.path.with.plugin:

build.project.class.path:

transform.merged.common.jaxp:
     [echo] Stylesheet=D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf5.ml\xsl\dita2fo_convmerged_shell.xsl
     [echo] use.i18n.index.lib=no
     [echo] user.input.dir=D:\SVN\pdf5\testdata\20200514-track-change
     [xslt] Processing C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\bmTrackChanges3_MERGED.xml to C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\bmTrackChanges3_CONVERTED.xml
     [xslt] Loading stylesheet D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf5.ml\xsl\dita2fo_convmerged_shell.xsl
     [xslt] Failed to process C:\Users\toshi\AppData\Local\Temp\temp20200819012120366\bmTrackChanges3_MERGED.xml
Error: The following error occurred while executing this line:
D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf5.ml\build.xml:207: The following error occurred while executing this line:
D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf5.ml\build_transform.xml:78: The following error occurred while executing this line:
D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf5.ml\build_transform.xml:106: The following error occurred while executing this line:
D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf5.ml\build_transform.xml:131: The following error occurred while executing this line:
D:\DITA-OT\dita-ot-3.5.2\plugins\com.antennahouse.pdf5.ml\build_transform.xml:143: java.lang.reflect.InvocationTargetException
PS D:\DITA-OT\dita-ot-3.5.2>

The catalog file processing seems to be not working correct.

         at org.apache.xml.resolver.tools.CatalogResolver.resolve(CatalogResolver.java:266)
        ... 129 more
    Caused by: java.net.MalformedURLException: unknown protocol: plugin
        at java.net.URL.<init>(Unknown Source)
        at java.net.URL.<init>(Unknown Source)
        at org.apache.xml.resolver.tools.CatalogResolver.resolve(CatalogResolver.java:256)

--- a/com.antennahouse.pdf5.ml/build.xml
+++ b/com.antennahouse.pdf5.ml/build.xml
@@ -188,6 +188,7 @@
     </condition>
 
     <!-- Make URL for input directory -->
+    <dirname property="user.input.dir" file="${args.input}"/>
     <makeurl property="user.input.dir.url" file="${user.input.dir}"/>
     
     <!-- Make URL for output directory -->

--- a/com.antennahouse.pdf5.ml/build.xml
+++ b/com.antennahouse.pdf5.ml/build.xml
@@ -51,7 +51,7 @@
     
     <!-- XML catalog -->
     <xmlcatalog id="dita.catalog">
-        <catalogpath path="${dita.dir}/catalog-dita.xml"/>
+        <catalogpath path="${dita.dir}${file.separator}plugins${file.separator}org.dita.base${file.separator}catalog-dita.xml"/>
     </xmlcatalog>
     
     <!-- Logical check -->

--- a/com.antennahouse.pdf5.ml/plugin.xml
+++ b/com.antennahouse.pdf5.ml/plugin.xml
@@ -146,6 +146,8 @@
       <val default="true">yes</val>
       <val>no</val>
     </param>
+    <param name="ahf.dir" desc="Specifies AH Formatter directory" type="string"/>
+    <param name="ahf.opt" desc="Specifies AH Formatter option setting file" type="file"/>
   </transtype>
   <template file="build_transform_template.xml"/>
   <template file="xsl/dita2fo_shell_template.xsl"/>

